### PR TITLE
Hacker news example

### PR DESCRIPTION
**Pull request by jvidalv**
# Description
Live example:
https://grafbase-nextjs-hackernews-with-clerk.vercel.app/

- [X] Explanation steps and how it works.
- [X] Pagination of items, users and posts.
- [X] Uncomment delete (It brokes the app, as it also deletes the linked users).

# Type of change

- [X] 🚀 Feature

### CODE REVIEW

**Comment by jvidalv** [comment]
This should be ready for review @vitalikda @notrab.

File: examples/nextjs-clerk-hacker-news/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+To understand how to connect Clerk as your Identity Provider with your Grafbase project &mdash; [Read the guide](https://grafbase.com/guides/using-clerk-as-your-identity-provider-with-grafbase)
+
+## Getting Started
+
+1. Run `npx degit grafbase/grafbase/examples/nextjs-clerk-hacker-news grafnews-with-clerk` to clone this example
+2. Change directory into the new folder `cd grafnews-with-clerk`
+3. Run `cp .env.example .env` to copy the example `.env.example` file to `.env` in the root of your project
+4. Run `cp grafbase/.env.example grafbase/.env` to copy the example `.env.example` file to `.env` in the `grafbase` folder
+5. Open `.env` in your code editor, and provide your Grafbase API endpoint and [Clerk frontend and backend API keys](https://dashboard.clerk.dev/last-active?path=api-keys)
+6. Open `grafbase/.env` in your code editor, and provide your Clerk issuer URL
```
Comment (vitalikda) [inline examples/nextjs-clerk-hacker-news/README.md:14]: CLI requires url protocol - http/https, should we mention this?

File: examples/nextjs-clerk-hacker-news/pages/index.tsx [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+      </Head>
+      <div className="space-y-8">
+        <div className="bg-indigo-600 p-4 border border-b-4 border-indigo-300 space-y-4 sm:space-y-0 sm:flex items-center justify-between sm:space-x-4">
+          <h2 className="text-white text-2xl ">
+            Grafbase is open to everyone, start building your frontend with the
+            next-gen GraphQL platform!
+          </h2>
+          <div>
+            <a
+              href="https://grafbase.com/register"
```
Comment (vitalikda) [inline examples/nextjs-clerk-hacker-news/pages/index.tsx:76]: Outdated: path no longer exists 
Also, I would add target blank

Comment (jvidalv) [reply examples/nextjs-clerk-hacker-news/pages/index.tsx:76]: It does exist, its a path to our website: https://grafbase.com/register. 

Yes, sure!

File: examples/nextjs-clerk-hacker-news/README.md (hunk 1 above)
Comment (jvidalv) [reply examples/nextjs-clerk-hacker-news/README.md:14]: I'm not following, what do you mean exactly?

File: examples/nextjs-clerk-hacker-news/pages/index.tsx (hunk 2 above)
Comment (notrab) [reply examples/nextjs-clerk-hacker-news/pages/index.tsx:76]: Maybe we add a redirect on our website? It seems right now we show the same page at `/sign-up` and `/register`?

Comment (fbjork) [reply examples/nextjs-clerk-hacker-news/pages/index.tsx:76]: It should indeed be a permanent redirect

Comment (jvidalv) [reply examples/nextjs-clerk-hacker-news/pages/index.tsx:76]: It's a rewrite right now in next.config.

image


I don't see anything wrong with this, but could be a redirect, yes.

Comment (notrab) [reply examples/nextjs-clerk-hacker-news/pages/index.tsx:76]: It should be a 301 redirect.

File: examples/nextjs-clerk-hacker-news/README.md (hunk 1 above)
Comment (vitalikda) [reply examples/nextjs-clerk-hacker-news/README.md:14]: When I copy issuer url it doesn't include https, but grafbase cli requires it

File: examples/nextjs-clerk-hacker-news/pages/index.tsx (hunk 2 above)
Comment (vitalikda) [reply examples/nextjs-clerk-hacker-news/pages/index.tsx:76]: Sorry, when I opened the link I saw 404 that why I mentioned it

**Comment by jvidalv** [comment]
Have been updated and should work with both SSR and CSR @grafbase/frontend

**Review by vitalikda — APPROVED** [review]
Awesome example! 🔥

**Pull request merged**